src/serviceFacades/instantlaunches.js

```diff
@@ -221,6 +244,31 @@ export const removeFromDashboardHandler = async (id) => {
     return new Promise((resolve, reject) => resolve(ilMeta));
 };
 
+/**
+ * Removes an instant launch from the navigation drawer.
+ *
+ * @param {string} id - The UUID of the instant launch that should be removed from the dashboard.
+ */
+export const removeFromNavDrawer = async (id) => {
+    const ilMeta = await getInstantLaunchMetadata(id);
+
+    if (!ilMeta.avus) {
+        throw new Error("no avus in response");
+    }
+
+    const filtered = ilMeta.avus.filter(
+        ({ attr, value }) =>
+            attr !== instantLaunchLocationAttr ||
+            value !== instantLaunchNavDrawer
+    );
+
+    if (filtered.length > 0) {
+        return await resetInstantLaunchMetadata(id, filtered);
+    }
+
+    return new Promise((resolve, reject) => resolve(ilMeta));
+};
+
```

Comment:
While this works, I wonder if we can simplify (and standardize) by using react-queries - https://react-query.tanstack.com/guides/dependent-queries#_top
Same for `addToNavDrawer` function as well.

Reply:
I agree - I saw some opportunities to refactor for consistency with what we're doing in other components, but I also wanted to keep the changes short for this PR.  I can definitely do some refactoring in a separate PR.